certified-connectors/InEight/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "CLIENTID",
```

Comment:
@InEightDevOps Can you please make it to `"[[DUMMY]]"` to maintain consistency with other submissions.